system-parachains/people/people-polkadot/src/bridge_to_bulletin_config.rs

```diff
@@ -0,0 +1,280 @@
+	XcmRouter,
+};
+use bp_messages::{
+	source_chain::FromBridgedChainMessagesDeliveryProof, target_chain::FromBridgedChainMessagesProof, LegacyLaneId,
+};
+
+use frame_support::{
+	parameter_types,
+	traits::{Equals, PalletInfoAccess},
+};
+use frame_system::{EnsureNever, EnsureRoot};
+use pallet_bridge_messages::LaneIdOf;
+use pallet_bridge_relayers::extension::{
+	BridgeRelayersTransactionExtension, WithMessagesExtensionConfig,
+};
+use pallet_xcm_bridge_hub::XcmAsPlainPayload;
+use polkadot_parachain_primitives::primitives::Sibling;
+use system_parachains_constants::polkadot::currency::UNITS as ROC;
+use xcm::{
+	latest::prelude::*,
+	prelude::{InteriorLocation, NetworkId},
+	AlwaysV5,
+};
+use xcm_builder::{BridgeBlobDispatcher, ParentIsPreset, SiblingParachainConvertsVia};
+
+parameter_types! {
+	/// Interior location (relative to this runtime) of the with-PolkadotBulletin messages pallet.
+	pub BridgePolkadotToPolkadotBulletinMessagesPalletInstance: InteriorLocation = [
+		PalletInstance(<BridgePolkadotBulletinMessages as PalletInfoAccess>::index() as u8)
+	].into();
+	/// Polkadot Bulletin Network identifier.
+	pub PolkadotBulletinGlobalConsensusNetwork: NetworkId = NetworkId::PolkadotBulletin;
+	/// Relative location of the Polkadot Bulletin chain.
+	pub PolkadotBulletinGlobalConsensusNetworkLocation: Location = Location::new(
+		2,
+		[GlobalConsensus(PolkadotBulletinGlobalConsensusNetwork::get())]
+	);
+
+	// see the `FEE_BOOST_PER_RELAY_HEADER` constant get the meaning of this value
+	pub PriorityBoostPerRelayHeader: u64 = 58_014_163_614_163;
+
+	/// Priority boost that the registered relayer receives for every additional message in the message
+	/// delivery transaction.
+	///
+	/// It is determined semi-automatically - see `FEE_BOOST_PER_MESSAGE` constant to get the
+	/// meaning of this value.
+	pub PriorityBoostPerMessage: u64 = 364_088_888_888_888;
+
+	/// PeoplePolkadot location
+	pub PeoplePolkadotLocation: Location = Location::new(1, [Parachain(polkadot_runtime_constants::system_parachain::PEOPLE_ID)]);
+
+	pub storage BridgeDeposit: Balance = 5 * ROC;
+}
+
+/// Proof of messages, coming from Polkadot Bulletin chain.
+pub type FromPolkadotBulletinMessagesProof<MI> =
+	FromBridgedChainMessagesProof<bp_polkadot_bulletin::Hash, LaneIdOf<Runtime, MI>>;
+/// Messages delivery proof for Polkadot Bridge Hub -> Polkadot Bulletin messages.
+pub type ToPolkadotBulletinMessagesDeliveryProof<MI> =
+	FromBridgedChainMessagesDeliveryProof<bp_polkadot_bulletin::Hash, LaneIdOf<Runtime, MI>>;
+
+/// Dispatches received XCM messages from other bridge.
+type FromPolkadotBulletinMessageBlobDispatcher = BridgeBlobDispatcher<
+	XcmRouter,
+	UniversalLocation,
+	BridgePolkadotToPolkadotBulletinMessagesPalletInstance,
+>;
+
+/// Transaction extension that refunds relayers that are delivering messages from the Polkadot
+/// Bulletin chain.
+pub type OnPeopleHubPolkadotRefundPolkadotBulletinMessages = BridgeRelayersTransactionExtension<
```

Comment:
@antkve not sure if we need this, at least add please some TODO or issue to check rewards at the end